index.html

```diff
@@ -128,6 +128,26 @@ <h4>Suzaan clarke</h4>
 </section>
 <!-- End Our Experts Section -->
 
+<!-- Start Contact us Section -->
+<section id="contact-us">
+  <div class="container">
+      <div class="title"><h2>Contact Us</h2></div>
+      <form>
+          <input type="text" id="name" class="field" placeholder="Your Name">
+          <input type="text" id="email" class="field" placeholder="Your Email">
+          <textarea id="message" placeholder="Your Message" class="field"></textarea>
+          <button class="submitBtn">Submit</button>
+      </form>
+  </div>
+</section>
+
+<div class="messageThank">
```

Comment:
make the name as your convention for CSS which is snake-case

Reply:
Add a style for the pop up message to make it smaller

Reply:
Do you mean like that 'message_thank' ?